inference-engine/tests/functional/inference_engine/ngraph_reader/proposal_tests.cpp

```diff
@@ -32,9 +32,9 @@ TEST_F(NGraphReaderTests, ReadProposalNetwork) {
             </output>
         </layer>
         <layer id="2" name="in3" type="Const" version="opset1">
-            <data element_type="i64" offset="0" shape="3" size="24"/>
```

Comment:
Why did you change the test? Do we break the old behavior?

Reply:
Well, the spec states that all inputs for this op are of the same, floating point type.  I just added checks in `validate_and_infer_types`, that pointed out that this should not work.

Reply:
@lazarevevgeny I think it can break the backward compatibility. What do you think?

Reply:
The inputs should not be of I64 data type for sure. All inputs should be of floating point type. Theoretically, they can be of different precisions: FP32 and FP16 (for case of Myriad plugin).
But anyway, there should be no backward compatibility issues with that.